### ISSUE TITLE
A11y issue with searchable select 

### ISSUE THREAD

**Issue by ElenaShevchenko** [JedWatson/react-select#5223]
I use NVDA with Chrome for testing
There are two issues 
1) When I try to search something which is not match any  option 
For example I try to search "Hello" here https://react-select.com/async (first example) 
"No options" will appear and visually it is ok, but Screen Reader say nothing in this case, but it should say "no options" or maybe "number of available results is zero" or something to notify user that there is np results
2) If I try to search something which match an option with the first letter, for example "O" - SR will say correct number of option,  but if then I will continue typing and type "Odg"  SR again repeat the previous result which is not correct

**Comment by AleksandraRataj** [comment]
Hi! 
I came across the same accessibility problem.  Although "No options" is displayed visually, it cannot be navigated using the arrow keys with a screen reader enabled. This could cause problems for some users, including partially sighted screen reader users, who may see the text but will not be able to hear it announced.
